clients/extension/scripts/extension-entry.js

```diff
@@ -5,133 +5,46 @@
  */
 'use strict';
 
-const lighthouse = require('../../../lighthouse-core/index.js');
-const Config = require('../../../lighthouse-core/config/config.js');
-const defaultConfig = require('../../../lighthouse-core/config/default-config.js');
-const i18n = require('../../../lighthouse-core/lib/i18n/i18n.js');
-
-const ExtensionProtocol = require('../../../lighthouse-core/gather/connections/extension.js');
-const log = require('lighthouse-logger');
+const defaultCategories = [{
```

Comment:
i like how we don't have to pull in config etc for all this.... but let's add a test that verifies that these categories match the categories in default-config

Reply:
done

Reply:
> i like how we don't have to pull in config etc for all this.... but let's add a test that verifies that these categories match the categories in default-config

+1 to the test, but I'm not sure what it saves either way. 36KB? Either way we'll have to remember to manually push a new version of the extension if the categories change

Reply:
It saves 692KB.

```
const Config = require('../../../lighthouse-core/config/config.js');
const defaultConfig = require('../../../lighthouse-core/config/default-config.js');
const i18n = require('../../../lighthouse-core/lib/i18n/i18n.js');

// ~692kb of deps
```

Reply:
> It saves 692KB.

it doesn't need to be a `Config`, just a `Config.Json`. Only `default-config.js` is needed

Reply:
would need to pull in i18n to translate the default config category title into a user string.

Reply:
even just i18n now makes the bundle go from 12K -> 5.1M. Something must have changed that recently ..

Reply:
I must have calculated the size incorrectly earlier - it was never just 692KB.

5.0.0:
```
> yarn browserify lighthouse-core/lib/i18n/i18n.js | wc -c | numfmt --to=iec
4.1M
```

master (77e57c91b)
```
> yarn browserify lighthouse-core/lib/i18n/i18n.js | wc -c | numfmt --to=iec
5.1M
```

Reply:
Let's consider i18n in the extension in a followup. (Or, preferably, not at all.)